### DISCUSSION TITLE
DietPi-Software | Pterodactyl as a mc panel

### DISCUSSION THREAD

**Discussion by Ast3risk-ops** [MichaIng/DietPi#5777] [Software Request]
#### Formal software information
- Software title |  Pterodactyl
- Short description | A free, open-source game server management panel built with PHP, React, and Go. 
- Official URL (if available) |  https://pterodactyl.io/ 
- GitHub URL (if available) |  https://github.com/pterodactyl 
- (Official) install documentation |  https://pterodactyl.io/panel/1.0/getting_started.html#picking-a-server-os 

#### Are there similar/alternative software titles available with DietPi-Software?

- MineOS and Cuberite although Cuberite doesn't support latest mc and MineOS doesn't support the latest UNIX user encryption.

#### What makes your requested software better than the above solutions, if available?

- It is easy to set up, looks great, has a lot of functionality and can be used for many games.

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?

- Installing dependencies and setting up the database and web interface, basically a lot of the steps in the docs. 

#### Can you provide the installation steps that you would suggest DietPi-Software to do?

1. Install dependencies (including Composer, which should be installed via Dietpi-software)
2. Set up webserver config, potentially with a self-signed SSL certificate.
3. Making a service for the panel and making it dietpi controlled.
4. Put it in \<your-ip\>/pterodactyl
5. Profit

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic :wink:.

- Not really, I have other things to do.

**Comment by Minecon724** [comment]
anybody?
